lib/trino-parquet/src/main/java/io/trino/parquet/writer/ParquetWriterOptions.java

```diff
@@ -21,6 +21,9 @@ public class ParquetWriterOptions
 {
     private static final DataSize DEFAULT_MAX_ROW_GROUP_SIZE = DataSize.ofBytes(ParquetWriter.DEFAULT_BLOCK_SIZE);
     private static final DataSize DEFAULT_MAX_PAGE_SIZE = DataSize.ofBytes(ParquetWriter.DEFAULT_PAGE_SIZE);
+    // org.apache.parquet.column.DEFAULT_PAGE_ROW_COUNT_LIMIT is 20_000 to improve selectivity of page indexes
+    // This value should be revisited when TODO https://github.com/trinodb/trino/issues/9359 is implemented
+    public static final int DEFAULT_MAX_PAGE_VALUE_COUNT = 60_000;
```

Comment:
I guess lower is better for the reader, right? Why not 20_000 then?

Reply:
It depends, lower value helps more when there are page level indexes (we don't write those yet). Lower value reduces peak memory usage (especially when there are a lot of columns being read) and more opportunities for skipping decompression of pages when there is filtering due to lazy block loading. Higher value is potentially better for simple scans as it decompresses more data in one go. Higher value might also give better compression ratios due to trying to pack more values per page.